about_us.html

```diff
@@ -82,29 +82,60 @@
                             vertical
                             class="description-bg">
                     <v-carousel-item v-for="(slide, i) in slides"
-                                     :key="i">
+                                        :key="i">
                         <v-sheet height="100%">
-                            <v-row>
-                                <div class="name">
-                                    {{slide.name}}
-                                </div>
-                            </v-row>
-                            <v-row>
-                                <div class="desc-title">
-                                    {{slide.title}}
-                                </div>
-                            </v-row>
-                            <v-row>
-                                <div class="description">
-                                    {{slide.description}}
-                                </div>
-                            </v-row>
-                            <v-row>
-                                <div class="skills">
-                                    <ul v-for="skill in slide.skills">
-                                        <li>{{skill.name}} : {{skill.power_level}}</li>
-                                    </ul>
-                                </div>
+                            <v-row class="description-row">
+                                <v-col class="description-sizer" cols="12" md="6" style="text-align: left;">
+                                    <div class="name">
+                                        {{slide.name}}
+                                    </div>
+                                    <div class="desc-title">
+                                        {{slide.title}}
+                                    </div>
+                                    <div class="desc-title">
+                                        Top Skills:
+                                    </div>
+                                    <div class="skills">
+                                        <ul style="list-style-type: none">
+                                            <li v-for="skill in slide.skills"><b>{{skill.type}}</b>{{skill.skills}}</li>
```

Comment:
```suggestion
                                            <li v-for="skill in slide.skills"><strong>{{skill.type}}</strong>{{skill.skills}}</li>
```
<strong> tag is preferred to <b> tag now.